src/server/bitops_family.cc

```diff
@@ -355,7 +355,7 @@ OpResult<bool> BitNewValue(const OpArgs& args, std::string_view key, uint32_t of
                            bool bit_value) {
   EngineShard* shard = args.shard;
   ElementAccess element_access{key, args};
-  auto& db_slice = shard->db_slice();
+  auto& db_slice = args.db_cntx.ns->GetCurrentDbSlice();
```

Comment:
similarly, please add a convenience accessor `GetDbSlice` in OpArgs that uses the shard variable inside OpArgs

Reply:
Adding it in OpArgs is a great idea, it saves a lot of boilerplate, thanks!